yellowbrick/regressor/residuals.py

```diff
@@ -504,25 +526,27 @@ def draw(self, y_pred, residuals, train=False, **kwargs):
 
         if train:
             color = self.colors['train_point']
-            alpha = 0.5
             label = "Train $R^2 = {:0.3f}$".format(self.train_score_)
         else:
             color = self.colors['test_point']
-            alpha = 0.9
```

Comment:
I'm not sure what to recommend here, but I do believe it's important to ensure that train/test have different alpha values - there are usually far more train points than test points, and without the increase in opacity it is hard to see the test points. Perhaps we could do something similar to what you did for `colors` - have `alphas["train"]` and `alphas["test"]`?

Reply:
True, though it might not be *quite* as bad as you might think. Here are some samples with different values of alpha (applied equally to a train of 80% and a test of 20%):

(alpha = 0.9)
![image](https://user-images.githubusercontent.com/8760385/44630982-a7e48e00-a933-11e8-98f0-67afce151511.png)

(alpha = 0.7)
![image](https://user-images.githubusercontent.com/8760385/44630978-9f8c5300-a933-11e8-89bc-ca818f6b4856.png)

(alpha = 0.5)
![image](https://user-images.githubusercontent.com/8760385/44630975-8e434680-a933-11e8-87ba-0149dddc154b.png)

(alpha = 0.3)
![image](https://user-images.githubusercontent.com/8760385/44630977-969b8180-a933-11e8-8472-d627fe6a2525.png)

Reply:
You got it!